CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # ChangeLog
 
-## **next**
+## **5.99.0** (2024-09-30)
 
-### Enhancements
+<img id="latest" src="https://badgen.net/badge/channel/latest/yellow" alt="Channel: latest" />
+
+### Highlights
```

Comment:
all the enhancements/bugfixes are not highlights, please check the items marked as highlight in project.vates.tech